superset-frontend/src/components/Select/Select.tsx

```diff
@@ -165,6 +165,11 @@ export interface SelectProps extends PickedSelectProps {
    * Will not apply to predefined `options` array when users are not searching.
    */
   sortComparator?: typeof DEFAULT_SORT_COMPARATOR;
+  /**
+   * Customize who the container node is.
+   * Parent node by default.
+   */
+  getPopupContainer?: (triggerNode?: any) => any;
```

Comment:
I think the return should be `HTMLElement`

Reply:
Yeah, I had it like that, but SelectFilterPlugin is also customizing such prop and I didn't want to impact other non-related component.